components/pedigree/resources/src/main/resources/pedigree/pedigreeDate.js

```diff
@@ -146,7 +146,7 @@ define([], function(){
 
         /** Returns true iff the minimum precision level is set, ie. the year. Useful for displaying the date. */
         isComplete: function() {
-            return (this.year !== null);
+            return (this.year !== null && (!this.range || this.range.years == 1));
```

Comment:
Good you reverted this, the fact that range is "1 year" does not mean the date is "complete" in the sense we know it with at least some precision (e.g. `range == 1` and `day == `1 but `month == null && year == null`)

Reply:
Yep, I had misunderstood and figured out the mistake when birth years stopped displaying.